packages/plugin-commands-installation/src/update.ts

```diff
@@ -104,6 +111,11 @@ export function help () {
               dependencies is not found inside the workspace`,
```

Comment:
```suggestion
              dependencies is not found inside the workspace.`,
```

Reply:
I am trying to be consistent. No dot at the end of help descriptions.  If the dot is needed, then it should be added to each description

Reply:
The rule I always use is:

- Single sentence, no period
- Multiple sentences, put a period on the last sentence. Otherwise it's **inconsistent** within the description and looks weird.

Reply:
ok, I am fine with this rule. We can change the help descriptions in a different PR.